### PR TITLE
Add api docs

### PR DESCRIPTION
**Pull request by pcarranza**

### CODE REVIEW

File: content/api.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+* `token-new @user #channel command [args...]`
+
+This invokation will then return a UUID which is the API token. This UUID will
+be linked to the command with the proposed arguments. The command will be
+executed as the @user in the #channel.
+
+Permissions-wise, the command access will be evaluated when it is being
+invoked, which means that an admin can create a token for a user who does not
+have access to the requested command, or the user can loose access to the
```
Comment (omame) [inline content/api.md]: `s/loose/lose/`?

File: content/api.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+they would be leaking to any user who has access the used channel. Any attempt
+to invoke any token command in an open channel will be rejected by the
+meesseeks.
+
+## Creating an api token
+
+* `token-new @user #channel command [args...]`
+
+This invokation will then return a UUID which is the API token. This UUID will
+be linked to the command with the proposed arguments. The command will be
```
Comment (omame) [inline content/api.md:22]: Can the command be an alias?

File: content/api.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+executed as the @user in the #channel.
+
+Permissions-wise, the command access will be evaluated when it is being
+invoked, which means that an admin can create a token for a user who does not
+have access to the requested command, or the user can loose access to the
+command afterwards, in both cases, the token can be used, it will simply be
```
Comment (omame) [inline content/api.md]: Too many commas. How about this?
>Permissions-wise, the command access level will be evaluated when it is being invoked. This means that an admin can create a token for a user who does not have access to the requested command, or the user can lose access to the command afterwards. In both cases, the token can be used but the command will simply be rejected at execution time, resulting in an error in the registered `#channel`.

File: content/api.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+rejected in execution time, resulting in an error in the registered #channel.
+
+## Using an API token
+
+To use a created token, you should invoke the http api endpoint with a HTTP
+POST method. For example, using curl:
+
+`curl -X POST -H "TOKEN: <UUID>" localhost:9696/message`
+
+By default the http port is 9696 and the meeseeks are listening on 0.0.0.0.
```
Comment (omame) [inline content/api.md]: I'd write
>By default meeseeks-box HTTP API listens on 0.0.0.0:9696.

File: content/api.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+Permissions-wise, the command access will be evaluated when it is being
+invoked, which means that an admin can create a token for a user who does not
+have access to the requested command, or the user can loose access to the
+command afterwards, in both cases, the token can be used, it will simply be
+rejected in execution time, resulting in an error in the registered #channel.
+
+## Using an API token
+
+To use a created token, you should invoke the http api endpoint with a HTTP
```
Comment (omame) [inline content/api.md]: I'd just drop "created".

File: content/api.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+This will return a list of the existing tokens with the following format:
+
+> - *UUID* @user at #channel _command args..._
+
+## Revoking api tokens
+
+* `token-revoke <UUID>`
+
+This will destroy the token and it will not be available anymore
```
Comment (omame) [inline content/api.md]: Missing closing `.`.

**Review by omame — CHANGES_REQUESTED** [review]

File: content/api.md (hunk 2 above)
Comment (pcarranza) [reply content/api.md:22]: yes, because it will be lazily evaluated.

Comment (pcarranza) [reply content/api.md:22]: This is why everything about commands happens inside commands, it's transparent for the rest of the layers.

**Pull request merged**